Test_carbonreceiver_EndToEnd is unstable

See https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector-contrib/5536/workflows/dac796c8-4e59-4cc3-9364-7e7cb4314f06/jobs/42228

```
--- FAIL: Test_carbonreceiver_EndToEnd (0.00s)
    --- FAIL: Test_carbonreceiver_EndToEnd/default_config_udp (0.00s)
        receiver_test.go:216: 
            	Error Trace:	receiver_test.go:216
            	Error:      	Received unexpected error:
            	            	listen udp 127.0.0.1:50248: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
            	Test:       	Test_carbonreceiver_EndToEnd/default_config_udp
FAIL
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver	0.060s
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/protocol	0.104s
--- FAIL: Test_Server_ListenAndServe (0.01s)
    --- FAIL: Test_Server_ListenAndServe/udp (0.00s)
        server_test.go:64: 
            	Error Trace:	server_test.go:64
            	Error:      	Received unexpected error:
            	            	listen udp 127.0.0.1:50252: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
            	Test:       	Test_Server_ListenAndServe/udp
FAIL
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/transport	0.049s
?   	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver/transport/client	[no test files]
FAIL
```

--- a/internal/common/testutil/testutil.go
+++ b/internal/common/testutil/testutil.go
@@ -85,11 +85,16 @@ func GetAvailablePort(t *testing.T) uint16 {
 
 // Get excluded ports on Windows from the command: netsh interface ipv4 show excludedportrange protocol=tcp
 func getExclusionsList(t *testing.T) []portpair {
-	cmd := exec.Command("netsh", "interface", "ipv4", "show", "excludedportrange", "protocol=tcp")
-	output, err := cmd.CombinedOutput()
-	require.NoError(t, err)
+	cmdTCP := exec.Command("netsh", "interface", "ipv4", "show", "excludedportrange", "protocol=tcp")
+	outputTCP, errTCP := cmdTCP.CombinedOutput()
+	require.NoError(t, errTCP)
+	exclusions := createExclusionsList(string(outputTCP), t)
 
-	exclusions := createExclusionsList(string(output), t)
+	cmdUDP := exec.Command("netsh", "interface", "ipv4", "show", "excludedportrange", "protocol=udp")
+	outputUDP, errUDP := cmdUDP.CombinedOutput()
+	require.NoError(t, errUDP)
+	exclusions = append(exclusions, createExclusionsList(string(outputUDP), t)...)
+
 	return exclusions
 }
 

--- a/receiver/jaegerreceiver/jaeger_agent_test.go
+++ b/receiver/jaegerreceiver/jaeger_agent_test.go
@@ -47,12 +47,7 @@ import (
 
 var jaegerAgent = config.NewComponentIDWithName(typeStr, "agent_test")
 
-var skip = func(t *testing.T, why string) {
-	t.Skip(why)
-}
-
 func TestJaegerAgentUDP_ThriftCompact(t *testing.T) {
-	skip(t, "Flaky Test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10368")
 	port := testutil.GetAvailablePort(t)
 	addrForClient := fmt.Sprintf(":%d", port)
 	testJaegerAgent(t, addrForClient, &configuration{
@@ -76,7 +71,6 @@ func TestJaegerAgentUDP_ThriftCompact_InvalidPort(t *testing.T) {
 }
 
 func TestJaegerAgentUDP_ThriftBinary(t *testing.T) {
-	skip(t, "Flaky Test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10369")
 	port := testutil.GetAvailablePort(t)
 	addrForClient := fmt.Sprintf(":%d", port)
 	testJaegerAgent(t, addrForClient, &configuration{
@@ -86,7 +80,6 @@ func TestJaegerAgentUDP_ThriftBinary(t *testing.T) {
 }
 
 func TestJaegerAgentUDP_ThriftBinary_PortInUse(t *testing.T) {
-	skip(t, "Flaky Test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10370")
 	// This test confirms that the thrift binary port is opened correctly.  This is all we can test at the moment.  See above.
 	port := testutil.GetAvailablePort(t)
 

--- a/receiver/statsdreceiver/receiver_test.go
+++ b/receiver/statsdreceiver/receiver_test.go
@@ -19,7 +19,6 @@ import (
 	"context"
 	"errors"
 	"net"
-	"runtime"
 	"strconv"
 	"testing"
 	"time"
@@ -94,9 +93,6 @@ func TestStatsdReceiver_Flush(t *testing.T) {
 }
 
 func Test_statsdreceiver_EndToEnd(t *testing.T) {
-	if runtime.GOOS == "windows" {
-		t.Skip("skipping test on windows, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10151")
-	}
 	addr := testutil.GetAvailableLocalAddress(t)
 	host, portStr, err := net.SplitHostPort(addr)
 	require.NoError(t, err)

